Providers that can use compute profiles in Provisioning (#2251)
theforeman/foreman-documentation: guides/common/modules/proc_creating-compute-profiles.adoc

* `2-Medium`
* `3-Large`

> You can apply the compute profiles to any supported compute resource:

Reviewer: ```suggestion
You can apply compute profiles to all supported compute resources:
```

* `2-Medium`
* `3-Large`

You can apply the compute profiles to the following compute resource providers:

> * VMware

Reviewer 2: Question:
Is there a difference between CRs supported in upstream and downstream?

Because complete list would include also the following:
* https://github.com/theforeman/foreman_google
* https://github.com/theforeman/foreman_azure_rm
* EC2 / AWS
* OpenStack

Reviewer: I can only speak for orcharhino: orcharhino does not support OpenStack.

Author: @stejskalleos Is it the same list for applying Compute Profiles as it is for provisioning in general?

Reviewer 2: Yes, every compute resource can have profiles, including bare metal which I forgot to list

Author: Really? How bare metal? Physical hosts have a given number of CPUs and memory, for what is a compute profile good there?

Reviewer 2: Yeah you are right, no CPs for bare-metal.
Thinking where I got the idea that it's also for bare metals.
Sorry!

* Libvirt

> * Red Hat Virtualization (deprecated)

Reviewer: ```suggestion
* {oVirt}
```

If you really want to say "deprecated", I'd suggest to use "if satellite" and "if not satellite". See https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L76 and https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-satellite.adoc#L76

Reviewer: Or maybe add "depreacted" to the attribute?

> 

Reviewer: IMHO this list is missing Proxmox. cc @Manisha15

Reviewer 3: Yes, proxmox should also be in the list.
